src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,230 @@
 package core.basesyntax;
 
+import java.util.Collection;
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the redundant empty line at the start of the class definition.